src/utils.cpp

```diff
@@ -144,6 +144,18 @@ std::string ToCIFS(std::string& UNCName)
   return CIFSname;
 }
 
+bool insertUser(std::string& UNCName)
+{
+  std::string SMBPrefix = "smb://" + g_szUser + ":" + g_szPass + "@";
```

Comment:
This hardcoded format implies that there are always both non-empty username and password. Is this the case? I don't think that `smb://:@host/foo/bar` is syntactically valid.

Reply:
Additionally, this string can be `const`.